nuxeo-web-ui-ftest/webdriver/test/pages/ui.js

```diff
@@ -30,6 +30,10 @@ export default class UI extends BasePage {
     return this.el.element('#createBtn');
   }
 
+  get adminButton() {
```

Comment:
No, we had the same problem about a year ago because `this.ui.drawer.administration` is also the administration menu, so I had to create a separate helper for the administration button.